client/app/vps/modal/shortcut/reinstall/vps-reinstall.less

```diff
@@ -0,0 +1,5 @@
+.vps-reinstall {
+    & select.ssh-keys {
+        overflow-y: scroll;
+    }
+}
```

Comment:
`insert_final_newline` (Does EditorConfig is correctly installed on your side?)

Reply:
yes it is, do you see this in all pages?